packages/db/schema.prisma

```diff
@@ -482,4 +483,15 @@ model ShortURL {
 
   @@index([userId])
   @@index([shortUrlSlug])
-}
\ No newline at end of file
+}
+
+model UserBadges {
```

Comment:
one moar, before we push the schema to staging can you make this `UserBadge` as tables are singular for all the others.

Reply:
my brother in christ, we might have already lost the battle
https://github.com/search?q=repo%3Atypehero%2Ftypehero+%22S+%7B%22+language%3APrisma&type=code&l=Prisma

Reply:
nah should be fine. betatokens can be removed and the rest are enums

Reply:
changed to UserBadge. Also found some lack of prisma usage, so updated the PR to use prisma more where it wasn't terribly difficult too.